### PR TITLE
added responsive 'our products' without slider

### PR DESCRIPTION
**Pull request by bmatiichak**
1. _listings.less was totally rewritten with usage of Magento media-queries for product responsiveness. I didn't do much about 4th column dropping into second row on tablet because soon it will go inside slider and there will be no second row.
2. product image size was changed in view.xml id="new_products_content_widget_grid" to 435x540, then it was limited by min-width for smaller screens.

full hd
tablet
mobile

### CODE REVIEW

File: app/design/frontend/Magento/elogic-theme/Magento_Catalog/web/css/source/module/_listings.less [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+    .block-content {
+        .products-grid {
+            margin-top: 30px;
+        }
+    }
+
+    .product-items {
+        .lib-vendor-prefix-display(flex);
+        .lib-vendor-prefix-flex-wrap(wrap);
+        justify-content: space-between;
```
Comment (y-mantuliak) [inline app/design/frontend/Magento/elogic-theme/Magento_Catalog/web/css/source/module/_listings.less]: just additional thing (`(justify-content)` mixin is missing in the `_utilities.less`):
create your own mixin with two parameters

File: app/design/frontend/Magento/elogic-theme/Magento_Catalog/web/css/source/module/_listings.less [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+
+    .product-item-info {
+        .lib-vendor-prefix-display(flex);
+        .lib-vendor-box-align(center);
+        .lib-vendor-prefix-flex-direction(column);
+        text-transform:uppercase;
+    }
+
+    .product-item-details {
+        margin: 30px 0;
```
Comment (y-mantuliak) [inline app/design/frontend/Magento/elogic-theme/Magento_Catalog/web/css/source/module/_listings.less]: please check `@indent__` variables

File: app/design/frontend/Magento/elogic-theme/Magento_Catalog/web/css/source/module/_listings.less [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+    .page-layout-1column .block.widget .block-content .products-grid .product-item {
+        width: calc((100% - 6%) / 4);
+    }
+}
+
+.media-width(@extremum, @break) when (@extremum = 'min') and (@break = @screen__lg) {
+    .page-layout-1column .block.widget .block-content .products-grid .product-item {
+        width: calc((100% - 6%) / 4);
+    }
+}
```
Comment (y-mantuliak) [inline app/design/frontend/Magento/elogic-theme/Magento_Catalog/web/css/source/module/_listings.less:219]: you can use nesting for `.product-item` class

File: app/design/frontend/Magento/elogic-theme/Magento_Cms/web/css/source/_module.less [hunk 4]
```diff
@@ -151,6 +151,70 @@
...
+
+// }
+
+// .media-width(@extremum, @break) when (@extremum = 'min') and (@break = @screen__m) {
+
+// }
+
+// .media-width(@extremum, @break) when (@extremum = 'min') and (@break = @screen__xl) {
+
+// }
```
Comment (y-mantuliak) [inline app/design/frontend/Magento/elogic-theme/Magento_Cms/web/css/source/_module.less]: don't leave commented code if you don't need it please

File: app/design/frontend/Magento/elogic-theme/etc/view.xml [hunk 5]
```diff
@@ -0,0 +1,316 @@
...
+        <item type="directory">Lib::css</item>
+        <item type="directory">Lib::lib</item>
+        <item type="directory">Lib::extjs</item>
+        <item type="directory">Lib::prototype</item>
+        <item type="directory">Lib::scriptaculous</item>
+        <item type="directory">Lib::less</item>
+        <item type="directory">Lib::mage/adminhtml</item>
+        <item type="directory">Lib::mage/backend</item>
+    </exclude>
+</view>
```
Comment (y-mantuliak) [inline app/design/frontend/Magento/elogic-theme/etc/view.xml:11]: Please remove everything you don't need to override

**Review by y-mantuliak — CHANGES_REQUESTED** [review]

File: app/design/frontend/Magento/elogic-theme/Magento_Catalog/web/css/source/module/_listings.less (hunk 3 above)
Comment (bmatiichak) [reply app/design/frontend/Magento/elogic-theme/Magento_Catalog/web/css/source/module/_listings.less:219]: Sorry, but do you mean visually nested code or what?
These classes are default magento classes for products widths + 1 class that i've added(.block-content) to add more specificity.

**Review by y-mantuliak — APPROVED** [review]

**Pull request merged**